cmd/ddev/cmd/config-global.go

```diff
@@ -318,7 +298,6 @@ func init() {
 	configGlobalCommand.Flags().String("xdebug-ide-location", "", "For less usual IDE locations specify where the IDE is running for Xdebug to reach it")
 	configGlobalCommand.Flags().Bool("use-traefik", true, "If true, use Traefik for ddev-router")
 	_ = configGlobalCommand.Flags().MarkDeprecated("use-traefik", "please use --router instead")
-	configGlobalCommand.Flags().String("router", globalconfigTypes.RouterTypeTraefik, fmt.Sprintf("Valid router types are %s, default is %s", strings.Join(globalconfigTypes.GetValidRouterTypes(), ", "), globalconfigTypes.RouterTypeDefault))
```

Comment:
I think we'd better mark the `--router` flag as deprecated, just like we did for `--use-traefik` above.
Or make it accept `traefik` only.